src/Api/User.hs

```diff
@@ -1,46 +1,41 @@
-{-# LANGUAGE DataKinds                  #-}
+{-# LANGUAGE DataKinds #-}
```

Comment:
Style nitpicks (I should really write these down in a CONTRIBUTING.md :smile: )

Please don't alter the formatting. `stylish-haskell` can be used to keep the import and language pragma formatting consistent.